pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -14,8 +14,8 @@ import (
 	"github.com/samalba/dockerclient"
```

Comment:
The branch was about adding volume support, not about replacing the use of "github.com/samalba/dockerclient" with apitype in create container.  I simply followed the pattern that was already in place in create container to also support the -v flag.
I suggest creating a separate pull request to replacing  "github.com/samalba/dockerclient" with apitype in create container and not mix apples with oranges.
--Ken